src/variables/integration/dropdown.spec.js

```diff
@@ -0,0 +1,20 @@
+describe("test dropdown menu", () => {
+  it("should open dropdown menu", () => {
+    cy.visit("localhost:3000");
+    cy.get(".gc-dropdown");
+    cy.get(".gc-dropdown__btn").click();
```

Comment:
don't EVER use styling classes for testing purposes, rather add dedicated testing classes `tst-dropdown-button` and get element with them.